rest-server/engines/src/main/scala/com/actionml/engines/urnavhinting/URNavHintingPreparator.scala

```diff
@@ -100,7 +100,7 @@ object URNavHintingPreparator extends LazyLogging with SparkMongoSupport {
         logger.info("EventName: " + eventName)
         // logger.info(s"first eventName is ${trainingData.actions.head._1.toString}")
         val ids = if (eventName == trainingData.actions.head._1.toString && trainingData.minEventsPerUser.nonEmpty) {
-          val dIDS = IndexedDatasetSpark(eventRDD, trainingData.minEventsPerUser.get)(sc)
+          val dIDS = IndexedDatasetSparkHelper(eventRDD, trainingData.minEventsPerUser.get)(sc)
```

Comment:
Huh? I was adding a constructor to IndexedDatasetSpark, a Mahout defined type.  Why change the name? It is not returning a "helper" it is returning an `IndexedDatasetSpark`